android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java

```diff
@@ -55,7 +55,8 @@
 
     private static final String MAIN_FRAGMENT_TAG = "mainFragment";
 
-    @Inject TBAStatusController mStatusController;
+    @Inject
+    TBAStatusController mStatusController;
```

Comment:
I just pushed a fix in https://github.com/the-blue-alliance/the-blue-alliance-android/commit/90e5e6642492ae86493a55948dcee8e4634394ab, I realized I forgot to set the property. Copy over the new file from master